packages/core/src/validate.js

```diff
@@ -15,6 +15,7 @@ function createAjvInstance() {
     multipleOfPrecision: 8,
     schemaId: "auto",
     unknownFormats: "ignore",
+    useDefaults: true,
```

Comment:
Why do this change? (it's a breaking change)

Reply:
To be honest with you as this was quite some time ago, I cannot remember why I have a feeling that this was me looking into things as it was not quite behaving as I was expecting i.e. when no value specified all things where essentially valid so would give multiple options (perhaps)

Reply:
Ok, let's try removing it and see if it still works.

Reply:
> To be honest with you as this was quite some time ago, I cannot remember why I have a feeling that this was me looking into things as it was not quite behaving as I was expecting i.e. when no value specified all things where essentially valid so would give multiple options (perhaps)

The reason is that ajv returns `true` when validating against a constant when the value is undefined (see your comments [here](https://github.com/rjsf-team/react-jsonschema-form/pull/1610#issuecomment-592485064) and [here](https://github.com/rjsf-team/react-jsonschema-form/pull/1610#discussion_r387647808)).